### ISSUE TITLE
krill 2019 size

### ISSUE THREAD

**Issue by pmcelhany** [NWFSC-OA-lab/krill_size_2019#1]
@dmperez89 can you please organize the krill_size_2019 repo into usual folders (R, data, output, figs). Also change file paths using the "here()" function so it is easy for other people to run the code.

**Comment by pmcelhany** [comment]
@dmperez89 please let me know when you have this cleaned up so we can look at the figures.

**Comment by pmcelhany** [comment]
@dmperez89 It look like you have done the tidy work on the krill_size_2019 repo. I'm going to work with it a bit this morning, so please don't make any edits until I let you know that I'm finished.

**Comment by dmperez89** [comment]
@pmcelhany yes, I organized into folders and was able to read in data in using "here." I ran into a problem writing out, though, so the code might still be a bit wonky. Enjoy swimming around in it.

**Comment by pmcelhany** [comment]
@dmperez89 The files I just pulled do not have the here function. Have you pushed your latest version?

**Comment by dmperez89** [comment]
@pmcelhany sorry about that, see if you have the here function to read in data now (the write is probably still broken)
